tests/src/test/java/alluxio/client/cli/fs/command/LsPathInAlluxioTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.client.cli.fs.command;
+
+import alluxio.client.cli.fs.AbstractFileSystemShellTest;
+import alluxio.client.file.FileSystemTestUtils;
+import alluxio.grpc.WritePType;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Tests for lsPathInAlluxio command.
+ */
+public class LsPathInAlluxioTest extends AbstractFileSystemShellTest {
+
+    @Test
+    public void lsInAlluxio() throws Exception {
+        FileSystemTestUtils
+                .createByteFile(sFileSystem, "/testRoot/testFileA", WritePType.MUST_CACHE, 50, 50);
+        FileSystemTestUtils
+                .createByteFile(sFileSystem, "/testRoot/testFileZ", WritePType.MUST_CACHE, 10, 10);
+        FileSystemTestUtils
+                .createByteFile(sFileSystem, "/testRoot/testLongFile", WritePType.MUST_CACHE, 100, 100);
+
+        String workerHost = sLocalAlluxioCluster.getWorkerAddress().getHost();
+        String expected = "";
+        String format = "%-25s %s\n";
+
+        sFsShell.run("lsInAlluxio", "/testRoot");
+        expected += String.format(format, "Worker Host Name", "In Alluxio");
+        expected += String.format(format, workerHost, 160);
+
+        sFsShell.run("lsInAlluxio", "-h", "/testRoot");
+        expected += String.format(format, "Worker Address", "In Alluxio");
+        expected += String.format(format, workerHost, "160B");
+
+        assertEquals(expected, mOutput.toString());
```

Comment:
Maybe `assertEquals` on each of them, with `-h` and without `-h`. Don't verify these two together.

Reply:
ok